templates/haproxy.cfg.j2

```diff
@@ -121,66 +121,68 @@ backend tls_{{ vhost.name }}
 {% endif %}
 
 {% if 'ocp' in haproxy_type %}
-frontend openshift-app-http
-    {% for bindip in haproxy_ocp_bindips %}
+{% for env in ocp_envs %}
+frontend "openshift-app-http-{{ env }}"
```

Comment:
if you have items in (haproxy)_ocp_envs, that means that you can then use env.name instead of just {{ env }}, and same things for other lines below